### PR TITLE
Adding TOTP profile

### PR DESCRIPTION
**Pull request by Kanapriya**

### CODE REVIEW

File: modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/totp-refresh.jag [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    var cookie = session.get('auth-cookie');
+    if(cookie == null){
+        var cookie = request.getParameter("cookie");
+        session.put('auth-cookie', cookie);
+    }
+    var userName = request.getParameter("user");
+    try{
+        log.info("Calling the web service .... url "+serverUrl);
+        var result = callWSReturnJSON(webService, serverUrl, "urn:refreshSecretKey", new Array(userName));
+        print(result);
```
Comment (hasinthaindrajee) [inline modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/totp-refresh.jag]: Can we remove these print statements ?

File: modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/totp-init.jag [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<%
```
Comment (hasinthaindrajee) [inline modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/totp-init.jag]: Please add license headers..

File: modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/edit-finish.jag [hunk 3]
```diff
@@ -1,4 +1,19 @@
 <%
+/*
+ * Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (cdwijayarathna) [inline modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/edit-finish.jag]: Format of license headers seems to be incorrect, they should contain the license year.

Comment (Kanapriya) [inline modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/edit-finish.jag]: Year has been added in following PR
https://github.com/wso2/product-is/pull/571

**Pull request merged**